### PR TITLE
BD-3177: Minor Liquid highlight edits

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-3177**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

**Review by colleenwoodford — APPROVED** [review]
no notes looks great!

File: _docs/_user_guide/personalization_and_dynamic_content/liquid.md [hunk 1]
```diff
@@ -73,12 +73,17 @@ The following new array and math filters are available for use in your Liquid as
...
+### Liquid updates
 
-Each Liquid element corresponds to a color, allowing you to differentiate your Liquid at-a-glance as your build your personalized messages. This appears in any Liquid editor.
+Each Liquid element corresponds to a color, allowing you to differentiate your Liquid at-a-glance in your Liquid editor.
 
 ![]({% image_buster /assets/img/liquid_color_code.png %})
 
+You can also leverage predicative Liquid for custom attributes, attribute names, and more as you build your personalized messages.
+
+![]({% image_buster /assets/img/liquid_auto_complete.gif %})
```
Comment (rachel-feinberg) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid.md:84]:
```suggestion
![]({% image_buster /assets/img/liquid_auto_complete.gif %}){: style="max-width=70%;"
```
Seems a bit big 😅

File: _docs/_user_guide/personalization_and_dynamic_content/liquid.md [hunk 2]
```diff
@@ -73,12 +73,17 @@ The following new array and math filters are available for use in your Liquid as
...
+### Liquid updates
 
-Each Liquid element corresponds to a color, allowing you to differentiate your Liquid at-a-glance as your build your personalized messages. This appears in any Liquid editor.
+Each Liquid element corresponds to a color, allowing you to differentiate your Liquid at-a-glance in your Liquid editor.
 
 ![]({% image_buster /assets/img/liquid_color_code.png %})
 
+You can also leverage predicative Liquid for custom attributes, attribute names, and more as you build your personalized messages.
+
+![]({% image_buster /assets/img/liquid_auto_complete.gif %}){: style="max-width=70%;"}
```
Comment (rachel-feinberg) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid.md:84]:
```suggestion
![]({% image_buster /assets/img/liquid_auto_complete.gif %}){: style="max-width:70%;"}
```

**Review by rachel-feinberg — APPROVED** [review]
Lgtm!

**Pull request merged**